kafka-webview-ui/src/main/java/org/sourcelab/kafka/webview/ui/manager/kafka/config/ClusterConfig.java

```diff
@@ -63,6 +64,7 @@
      */
     private ClusterConfig(
         final Set<String> brokerHosts,
+        final String defaultConsumerGroupId,
```

Comment:
Not sure if this belongs here, or in the ClientConfig, I need to continue playing around with the code a bit to determine the best place for this to live.